meta-mender-core/classes/mender-part-images.bbclass

```diff
@@ -43,6 +43,8 @@ IMAGE_NAME_SUFFIX = ""
 # Block storage
 ################################################################################
 
+MENDER_RFS_PART_EMPTY_B ??= "n"
```

Comment:
I would even go one step further, and say that this should be the default. I can't really see any good reason why partition B should be populated at all.

It is possible this will fail in our platform tests though, I'm not sure about the assumptions they make. Since this is pretty easy to test, I submitted a simpler version [here](https://github.com/mendersoftware/meta-mender/pull/1519). Let's see how it goes before proceeding with this.